app/design/adminhtml/default/default/template/catalog/product/edit/action/inventory.phtml

```diff
@@ -85,7 +85,7 @@ function toggleValueElementsWithCheckbox(checkbox) {
             </tr>
             <tr>
                 <td class="label"><label for="inventory_min_sale_qty"><?php echo Mage::helper('catalog')->__('Minimum Qty Allowed in Shopping Cart') ?></label></td>
-                <td class="value"><input type="text" class="input-text validate-number" id="inventory_min_sale_qty" name="<?php echo $this->getFieldSuffix() ?>[min_sale_qty]" value="<?php echo $this->getDefaultConfigValue('min_sale_qty')*1 ?>" disabled="disabled" />
+                <td class="value"><input type="text" class="input-text validate-number" id="inventory_min_sale_qty" name="<?php echo $this->getFieldSuffix() ?>[min_sale_qty]" value="<?php echo Mage::helper('catalog/product')->getDefaultProductValue('min_sale_qty', null) ?>" disabled="disabled" />
```

Comment:
Since `Mage::helper('catalog/product')->getDefaultProductValue('min_sale_qty', null)` always returns `Mage_Catalog_Helper_Product::DEFAULT_QTY`:

```suggestion
                <td class="value"><input type="text" class="input-text validate-number" id="inventory_min_sale_qty" name="<?php echo $this->getFieldSuffix() ?>[min_sale_qty]" value="<?php echo Mage_Catalog_Helper_Product::DEFAULT_QTY ?>" disabled="disabled" />
```

![image](https://user-images.githubusercontent.com/1106470/135985466-00552bd1-7f4a-4c9a-a96d-ca5200f00308.png)